[HCCDOC#1765]: Add recommendations-filtering content to Remote Health Monitoring (#70364)
openshift/openshift-docs: modules/filtering-insights-advisor-recommendations.adoc

// Module included in the following assemblies:
//
// * support/remote_health_monitoring/using-insights-to-identify-issues-with-your-cluster.adoc

:_mod-docs-content-type: PROCEDURE
[id="filtering-unnecessary-advisor-recommendations_{context}"]
= Filtering Insights advisor recommendations

> As an OpenShift cluster manager, you can filter the recommendations that are displayed on the recommendations list. By applying filters, you can reduce the number of reported recommendations and concentrate on your highest priority recommendations.

Author: Done

The following procedure demonstrates how to set and remove *Category* filters; however, the procedure is applicable to any of the filter types and respective values. 

.Prerequisites
You are logged in to the https://console.redhat.com/openshift[OpenShift Cluster Manager Hybrid Cloud Console].

.Procedure

> . Navigate to *Red Hat Hybrid Cloud Console* -> *OpenShift* -> *Advisor recommendations*.

Reviewer: ```suggestion
. Go to *Red Hat Hybrid Cloud Console* -> *OpenShift* -> *Advisor recommendations*.
```
Current [IBM Style word usage guidelines](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#n) say to use _navigate_ "only when a more direct verb, such as _click_, _select_, _browse_, or _go to_, is not applicable".

Author: Done

> . Navigate to the link:https://console.redhat.com/openshift/insights/advisor/recommendations[*Red Hat Hybrid Cloud Console* -> *OpenShift* -> *Advisor recommendations*] page.

Author: Done

> . Using the filter input fields above the list of recommendations, select the *Category* filter type.

Author: Done

> . Expand the right drop-down list and select the checkbox next to each category of recommendation you want to view. Leave the checkboxes for unnecessary categories clear.

Reviewer: I'm still wondering if there's some way to avoid referring to "left" and "right". [The IBM guidelines say not to refer to locations using sensory elements](https://www.ibm.com/docs/en/ibm-style?topic=elements-ui#location-of-interface-elements). Do these lists have a label or a heading of some kind that you can refer to?

Author: I've changed the first instance from "left filter drop-down list" to "main, filter-type drop-down list," and the second instance from "right drop-down list" to "filter-value drop-down list." It still sounds inelegant to me, but I'll take this to my team and ask how to make this better with respect to the ISG and your feedback.

> . Expand the right dropdown and check the box next to each category of recommendation you want to view. Leave unnecessary categories unchecked.

Reviewer: ```suggestion
. From the [add label name!] drop-down list, select each category of recommendation you want to view.
```

- [Per the ISG](https://www.ibm.com/docs/en/ibm-style?topic=elements-ui#location-of-interface-elements), "When you refer to the location of an element in a graphical user interface, do not use sensory words, such as _top_, _bottom_, _right_, or _left_." 
- Use [the ISG standards for drop-down lists](https://www.ibm.com/docs/en/ibm-style?topic=elements-ui#drop_down_list) whenever you refer to these UI elements.
- Per the ISG, do not use "check" to refer to marking a checkbox. Use "select".
- Also if it is a checkbox, refer to it as a "checkbox" and not a "box".

Author: Done. Just as an fyi, the dropdowns on the Hybrid Cloud Console filters don't have label names (or hover-over names); there are two of them, and they're side by side. We call them the primary and secondary filters in another Console project, but neither the engineers nor peer review like that phrasing so I'm trying to be specific for the users, given both the limitations and rapidly changing nature of the GUI.

> . Optionally, add additional filters to further refine the list.

Reviewer: ```suggestion
. Optional: Add additional filters to further refine the list.
```
Follow [the ISG guidance](https://www.ibm.com/docs/en/ibm-style?topic=format-procedures#indicating-optional-and-conditional-steps) for indicating optional steps.

Author: Done
